pubspec.yaml

```
@@ -28,6 +28,11 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
+  husky: ^0.1.6
+  lint_staged: ^0.4.0
+
+lint_staged:
+  '**/*.dart': dart format --fix && dart fix --apply
```

```suggestion
  '**/*.dart': dart format && dart fix --apply
```
Looking at the options available to `dart format`, --fix is not listed.

---

Done👍